mono_repo/lib/src/ci_shared.dart

```diff
@@ -9,11 +9,24 @@ import 'package_config.dart';
 import 'root_config.dart';
 import 'version.dart';
 
-final skipCreatedWithSentinel = Object();
+/// Object used to flag if code is running in a test.
+final testingZoneKey = Object();
```

Comment:
Normally this would be private, and we would expose a method to run a callback in a Zone with it set, not a huge deal though.

Reply:
I thought about that. But then the usage site gets a bit crazy. I'll ponder a bit more...

Reply:
yup better. fixed it.